File: java-vertx/src/main/java/io/swagger/server/api/model/V3RunResponse.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.server.api.model.V3Run;
import io.swagger.server.api.model.V3Status;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * V3RunResponse
 */

@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaVertXServerCodegen", date = "2024-04-14T03:35:56.953023300+10:00[Australia/Sydney]")



public class V3RunResponse   {
  private V3Run run = null;

  private V3Status status = null;

  public V3RunResponse run(V3Run run) {
    this.run = run;
    return this;
  }

  /**
   * Get run
   * @return run
   **/
    public V3Run getRun() {
    return run;
  }

  public void setRun(V3Run run) {
    this.run = run;
  }

  public V3RunResponse status(V3Status status) {
    this.status = status;
    return this;
  }

  /**
   * Get status
   * @return status
   **/
    public V3Status getStatus() {
    return status;
  }

  public void setStatus(V3Status status) {
    this.status = status;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    V3RunResponse v3RunResponse = (V3RunResponse) o;
    return Objects.equals(this.run, v3RunResponse.run) &&
        Objects.equals(this.status, v3RunResponse.status);
  }

  @Override
  public int hashCode() {
    return Objects.hash(run, status);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class V3RunResponse {\n");
    
    sb.append("    run: ").append(toIndentedString(run)).append("\n");
    sb.append("    status: ").append(toIndentedString(status)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: dart/lib/model/v3_run_response.dart
part of swagger.api;

class V3RunResponse {
  
  V3Run run = null;

  V3Status status = null;

  V3RunResponse();

  @override
  String toString() {
    return 'V3RunResponse[run=$run, status=$status, ]';
  }

  V3RunResponse.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    run = new V3Run.fromJson(json['run']);
    status = new V3Status.fromJson(json['status']);
  }

  Map<String, dynamic> toJson() {
    return {
      'run': run,
      'status': status
     };
  }

  static List<V3RunResponse> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3RunResponse>() : json.map((value) => new V3RunResponse.fromJson(value)).toList();
  }

  static Map<String, V3RunResponse> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3RunResponse>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3RunResponse.fromJson(value));
    }
    return map;
  }
}
